viam_cartographer.go

```diff
@@ -680,6 +689,21 @@ func (cartoSvc *CartographerService) DoCommand(ctx context.Context, req map[stri
 		return map[string]interface{}{postprocess.UndoCommand: SuccessMessage}, nil
 	}
 
+	if val, ok := req[postprocess.PathCommand]; ok {
+		path, ok := val.(string)
+		if !ok {
+			return nil, ErrBadPostprocessingPath
+		}
+
+		bytes, err := os.ReadFile(path)
```

Comment:
This is causing a lint issue because it isbad practice to readFile at a user specified path. How should we proceed?